src/App.tsx

```diff
@@ -63,6 +84,37 @@ const checkoutTheme = createTheme({
         },
       },
     },
+    MuiOutlinedInput: {
+      styleOverrides: {
+        root: {
+          borderRadius: "8px",
+        },
+      },
+    },
+    MuiLoadingButton: {
+      styleOverrides: {
+        loadingIndicator: {
+          color: "white !important",
+        },
+        root: ({ ownerState }) =>
+          (ownerState.loading && {
+            backgroundColor: "#0B3EE3 !important",
+            color: "transparent !important",
+          }) ||
+          (ownerState.disabled &&
```

Comment:
This should be taken up to the designers, but for this pr we can assume the button must stay grayed out when disabled, so you can just remove the (103-107) lines